config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/ingredients", to: "ingredients#index"
+  
+  get "/recipes/:id", to: "recipes#show"
+  post "/recipes", to: "recipes#create_ingredient"
```

Comment:
Similar to my comment in your controller. A more RESTful option could be
`post "/recipes/:recipe_id/ingredients", to: "recipe_ingredients#create"`